docs/datatypes/units.md

```diff
@@ -203,6 +203,26 @@ math.createUnit('θ', '1 rad')
 math.evaluate('1θ + 3 deg').toNumber('deg') // 60.29577951308232
 ```
 
+## Numeric type of the value of a unit
+
+The built-in units are always created with a value being a `number`. To turn the value into for example a `BigNumber` or `Fraction`, you can convert the value using the function `math.fraction` and `math.bignumber`:
+
+```js
+math.unit(math.fraction(10), 'inch').toNumeric('cm')  // Fraction 127/5
+math.fraction(math.unit(10, 'inch')).toNumeric('cm')  // Fraction 127/5
+
+math.bignumber(math.unit(10, 'inch')).toNumeric('cm') // BigNumber 25.4
+math.unit(math.bignumber(10), 'inch').toNumeric('cm') // BigNumber 25.4
+```
+
+When using the expression parser, it is possible to configure numeric values to be parsed as `Fraction` or `BigNumber`:
+
+```js
+math.config({ number: 'Fraction' })
+math.evaluate('10 inch').toNumeric('cm') // Fraction 127/5
+```
+
+
```

Comment:
What you think about adding the following use case ? 

_Beware that converting irrational numbers to it's fraction representation will lead to a loss of precision in comparison to it's numerical value_
```javascript 
const a = math.unit(math.pi);
const fa = math.fraction(a) 
const isEqual = fa.equals(a)) // it throws: loss of precision
```

Reply:
I think that throwing an error in this case is fine.

The idea behind it is: when some operation like `equals` _implicitly_ (internally, silently) converts a fraction and that results in a loss of precision, it should throw an error. When the user is OK with that, he/she can _explicitly_ convert the number into a fraction using `math.fraction` and that should not throw an error. Does that make sense?

Reply:
Yes , agreed :)

Reply:
Thanks for your feedback! I'll update the branch and merge it.